starter-code/src/functions-and-arrays.js

```diff
@@ -1,93 +1,488 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else {
+    return num2;
+  }
+}
```

Comment:
Here you could have also tried using the return Math.max(num1, num2).
This way you didnt need to check if one is bigger than the other and etc